Jellyfin.Plugin.PlaybackReporting/Api/UserActivityAPI.cs

```diff
@@ -226,16 +227,11 @@ public object Get(GetUserReport request)
             foreach(var user_info in report)
             {
                 string user_id = (string)user_info["user_id"];
-                string user_name = "Not Known";
-                bool has_image = false;
                 Guid user_guid = new Guid(user_id);
                 User user = _userManager.GetUserById(user_guid);
+                bool has_image = user?.ProfileImage is null ? false : true;
```

Comment:
```suggestion
                bool has_image = user?.ProfileImage != null;
```

Reply:
From my understanding from poking around the interwebz, `x is null` is preferred over `x == null`. I'm sure there's a better way to invert the result however.
In Python I would do `x is not None`.